ci/install_tcpreplay.bat

```diff
@@ -1,3 +1,3 @@
 git clone https://github.com/seladb/PcapPlusPlus-Deploy
-7z x PcapPlusPlus-Deploy\Packages\tcpreplay-4.4.1-win.zip
+7z x PcapPlusPlus-Deploy\Packages\tcpreplay-4.4.2-win.zip
```

Comment:
For Windows to work we need to add `tcpreplay` compiled to Windows to here: https://github.com/seladb/PcapPlusPlus-Deploy/tree/master/Packages